packages/ui/components/ui/tx/view/output.tsx

```diff
@@ -10,7 +10,7 @@ export const OutputViewComponent = ({ value }: { value: OutputView }) => {
       <ViewBox
         label='Output'
         visibleContent={
-          <div className='flex justify-between md:flex-col lg:flex-row'>
+          <div className='flex flex-col justify-between gap-2 sm:flex-row sm:gap-0'>
```

Comment:
This (and the similar change below) makes the address move to below the amount on small screens